### ISSUE TITLE
Allow For More Variance In Regards To The Scourge Invasion

### ISSUE THREAD

**Issue by usfshihy** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#350]
**Describe your suggestion in full detail below:**
I thought it was a bit disappointing that I was not able to change the path of the scourge initial invasion in Lordearon, whether I chose to play as the Highlord of Stratholme or The Lord Of Brill. It seems that it is built in to spread here and destroy those locations, I hoped perhaps I would've been able to save Stratholme before its too late or lead a resistance force akin to Lord Garithos's while the plague was beginning and so when the Scarlet Crusade showed up, it wouldn't be alone. I do hope you inspect such scenarios since they can offer a lot in terms of alternate history development.

**Comment by zumbak04** [comment]
> I thought it was a bit disappointing that I was not able to change the path of the scourge initial invasion in Lordearon, whether I chose to play as the Highlord of Stratholme or The Lord Of Brill. It seems that it is built in to spread here and destroy those locations, I hoped perhaps I would've been able to save Stratholme before its too late

Hello! Don't really get what you want. From the first look I thought you want so Ner'zhul can choose where the Plague hits. But after reaching this Highlord of Stratholme part, I'm confused since it's not up to Stratholme how the Plague acts.

**Comment by zumbak04** [comment]
Would like to tie it to https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/issues/44.

**Comment by zumbak04** [comment]
> From the first look I thought you want so Ner'zhul can choose where the Plague hits.

And note: I think it's a great idea, but will see if we can balance it.

**Comment by usfshihy** [comment]
Sorry if I had appeared all over the place hahaha. What I meant was that if you tried to play as Stratholme and the plague hits, you'll get no control on any of the next sequence of events, ie. whether you shut your gates off or not it won't matter. Arthas will enter and purge the city and your whole bloodline will cease to exist, implying that they all died in the purge, ending the game immediately. What I had hoped to do was to either spare Stratholme the purge or attempt to save that part of Lordaeron from falling entirely to the Scourge sort of like a Light's Hope scenario, a small location still under human control. I hope I was a bit clearer this time and thx for responding I didn't expect any sort of reply tbh!

**Comment by zumbak04** [comment]
> Sorry if I had appeared all over the place hahaha. What I meant was that if you tried to play as Stratholme and the plague hits, you'll get no control on any of the next sequence of events, ie. whether you shut your gates off or not it won't matter. Arthas will enter and purge the city and your whole bloodline will cease to exist, implying that they all died in the purge, ending the game immediately. What I had hoped to do was to either spare Stratholme the purge or attempt to save that part of Lordaeron from falling entirely to the Scourge sort of like a Light's Hope scenario, a small location still under human control. I hope I was a bit clearer this time and thx for responding I didn't expect any sort of reply tbh!

I see about the Plague, but if there will be an easy way to stop... it's MEEEH. Stopping Arthas may be an option for players indeed.

**Comment by robmart** [comment]
Allowing the player to stop the purge if they are ruler of stratholme is a good idea IMO. Straight up destroying the bloodline of the player is not a good idea.

**Comment by usfshihy** [comment]
I was not suggesting that the bloodline should be destroyed, I was pointing out that this is the case right now and yeah that its not a good idea.


Sent from Yahoo Mail for iPhone


On Tuesday, April 27, 2021, 1:58 AM, Robin ***@***.***> wrote:




Allowing the player to stop the purge if they are ruler of stratholme is a good idea IMO. Straight up destroying the bloodline of the player is not a good idea.

**Comment by robmart** [comment]
> 
> I was not suggesting that the bloodline should be destroyed, I was pointing out that this is the case right now unfortunately.

Yes, that is what I meant. The current bloodline destroying thing is bad and should be changed.

**Comment by robmart** [comment]
> 
> 
> > From the first look I thought you want so Ner'zhul can choose where the Plague hits.
> 
> And note: I think it's a great idea, but will see if we can balance it.

Opening new suggestion for that

**Comment by robmart** [comment]
@usfshihy I'm not really sure what you meant by the "The Lord Of Brill" part, could you expand on it?
